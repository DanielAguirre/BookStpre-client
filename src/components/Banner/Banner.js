import React from 'react';
// import './Banner.css';


const Banner = () => (
  <div className="banner">
    <h1 className="banner-title"> Bookstore</h1>
  </div>
);

module.exports = Banner;