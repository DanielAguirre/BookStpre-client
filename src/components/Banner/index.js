import React from 'react';
import glamorous from 'glamorous'

// import './Banner.css';

const Div = glamorous.div({
  width: '100%',
  height: '450px',
  overflow: 'hidden',
  color: 'white',
  backgroundImage: 'url(../img/header.jpg)',
});

const H1  = glamorous.h1 ({
  display: 'inline-block',
  margin: '200px 35%',
  fontSize: '55px',
});

module.exports = () => (
  <Div className="banner">
    <H1 className="banner-title">Bookstore</H1>
  </Div>
);
