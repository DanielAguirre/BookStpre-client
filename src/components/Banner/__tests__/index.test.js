import React from 'react';
import { shallow, mount } from 'enzyme';
import Banner from '../index';

it('<Banner />', () => {
  const component = shallow(<Banner/>);
  expect(Banner).toMatchSnapshot();
})