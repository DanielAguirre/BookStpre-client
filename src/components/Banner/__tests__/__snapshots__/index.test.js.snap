exports[`test <Banner /> 1`] = `[Function]`;
